apps/wallet-mobile/src/features/SetupWallet/useCases/SelectWalletFromList/WalletListItem.tsx

```diff
@@ -16,6 +20,28 @@ type Props = {
 export const WalletListItem = ({wallet, onPress}: Props) => {
   const {styles, colors} = useStyles()
   const {type} = getWalletItemMeta(wallet, colors)
+  const walletManager = useWalletManager()
```

Comment:
none of this is needed for now, is behind a feature flag, is just to show Product what is now capable of